tests/data/import.csv

```diff
@@ -0,0 +1,3 @@
+IBAN;Auszugsnummer;Buchungsdatum;Valutadatum;Umsatzzeit;Zahlungsreferenz;Waehrung;Betrag;Buchungstext;Umsatztext
+ATXYZ123400000;1;2017-02-14;2017-02-14;2017-02-14-11.57.44.235044;"";EUR;-100,00;"Bankomat";" BANKOMAT  00000483 K2 UM 11:56"

```

Comment:
I do not have such lines in the files I personally use, but we could come up with a simple example, like so:

```csv
IBAN;Auszugsnummer;Buchungsdatum;Valutadatum;Umsatzzeit;Zahlungsreferenz;Waehrung;Betrag;Buchungstext;Umsatztext;Saldo
ATXYZ123400000;1;2017-02-14;2017-02-14;2017-02-14-11.57.44.235044;"";EUR;-100,00;"Bankomat";" BANKOMAT  00000483 K2 UM 11:56";4.567,89
;;;;2017-02-14;;;;Kundenhinweis: Zinssatz auf 0,15% geändert;;
```

This example does two things:
- Add a column at the end for a balance, to be used for a final `balance` entry
- Add a line which can be parsed to a `note`